Documentation/Maintenance/Release.md

```diff
@@ -718,26 +726,12 @@ Doxygen HTML pages. Use the files to upload and create:
 Prior to the release, new `Remote` modules should be enabled in the Doxygen
 build's configuration.
 
-Run CMake in the binary build and enable `BUILD_DOXYGEN`, configure and
-generate, then:
-
-```bash
-cd Binaries/ITK
-make Documentation
-cd Utilities
-mv old_doxygen_directory DoxygenInsightToolkit-$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION
-tar -cf DoxygenInsightToolkit-$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION.tar DoxygenInsightToolkit-$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION
-gzip -9 DoxygenInsightToolkit-$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION.tar
-```
-
-Historical note: Before ITK 3.8, the documentation used to be generated in a
-directory called `Documentation/Doxygen`.
+Download the tarballs from the `latest` [ITKDoxygen Release] and rename the files to use
+the current version. Create a new [ITKDoxygen Release] with the files on a
+`vMAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION` tag.
```

Comment:
vMAJOR_VERSION -> v$MAJOR_VERSION @thewtex ?

Reply:
@jhlegarreta good catch, addressed in https://github.com/InsightSoftwareConsortium/ITK/pull/5145